doc/cephadm/services/rgw.rst

```diff
@@ -303,6 +306,21 @@ where the properties of this service specification are:
 * ``ssl_cert``:
     SSL certificate, if SSL is to be enabled. This must contain the both the certificate and
     private key blocks in .pem format.
+* ``use_keepalived_multicast``
+    Default is False. By default, cephadm will deploy keepalived config to use unicast IPs,
+    using the IPs of the hosts. The IPs chosen will be the same IPs cephadm uses to connect
+    to the machines. But if multicast is prefered, we can set ``use_keepalived_multicast``
+    to ``True`` and Keepalived will use multicast IP (224.0.0.18) to communicate between instances,
+    using the same interfaces as where the VIPs are.
+* ``vrrp_interface_network``
+    By default, cephadm will configure keepalived to use the same interface where the VIPs are
+    for VRRP communication. If another interface is needed, it can be set ``vrrp_interface_network``
+    with a networks to identify which ethernet interface to use.
+* ``first_virtual_router_id``
+    When deploying more than 1 ingress, this parameter can be used to ensure each keepalived will
+    have different virtual_router_id. In the case of using ``virtual_ips_list``, each IP will create
+    its own virtual router. So the first one will have ``first_virtual_router_id``, second one
+    will have ``first_virtual_router_id`` + 1, etc. Good values go from 0 to 255.
```

Comment:
**Valid** values go from **1** to 255.

Reply:
Maybe add default of 50 to this text, too.

Reply:
Done.